models/api-models.js

```diff
@@ -0,0 +1,7 @@
+const fs = require("fs/promises");
+
+exports.fetchDocs = () => {
+  return fs.readFile("endpoints.json", "utf8").then((data) => {
+    return JSON.parse(data);
+  });
+};
```

Comment:
Models are used to interact with the database, there is no database to talk to here. What is happening is that you are reading a json file, making that a string, parsing it back to an object and sending that to a controller which is sending the client back a json object...could that chain of events be made shorter? You can require the json object directly into both your controller and test file to make this simpler.